examples/browser/package.json

```diff
@@ -22,7 +22,7 @@
     "start:backend": "node ./src-gen/backend/main.js | bunyan",
     "start:backend:debug": "node ./src-gen/backend/main.js --loglevel=debug | bunyan",
     "start:frontend": "webpack-dev-server --open",
-    "test:ui": "wdio wdio.conf.js",
+    "test:ui": "concurrently --names backendfrontend,uitest --prefix \"[{name}]\" \"npm run start\" \"wdio --waitforTimeout 300000 wdio.conf.js\"",
```

Comment:
it is fine to change `package.json`, but don't forget to reflect them in `theia.package.json` since `package.json` is auto generated